Content.Server/Administration/Commands/TagCommands.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Content.Shared.Administration;
+using Content.Shared.Tag;
+using Robust.Shared.Console;
+
+namespace Content.Server.Administration.Commands
+{
+    [AdminCommand(AdminFlags.Debug)]
+    public sealed class AddTagCommand : LocalizedCommands
+    {
+        [Dependency] private readonly IEntityManager _entityManager = default!;
+
+        public override string Command => "addtag";
+        public override string Description => "Adds a tag to a given entity";
+        public override string Help => "Usage: addtag <entity uid> <tag>";
+
+        public override void Execute(IConsoleShell shell, string argStr, string[] args)
+        {
+            if (args.Length != 2)
+            {
+                shell.WriteError(Loc.GetString("shell-wrong-arguments-number"));
+                return;
+            }
+
+            if (!EntityUid.TryParse(args[0], out var entityUid))
+            {
+                shell.WriteError(Loc.GetString("shell-entity-uid-must-be-number"));
+                return;
+            }
+
+            if (!_entityManager.TrySystem(out TagSystem? tagSystem))
+                return;
+
+            if (tagSystem.TryAddTag(entityUid, args[1]))
+            {
+                shell.WriteLine($@"Added {args[1]} to {entityUid}.");
+            }
+            else
+            {
+                shell.WriteError($@"Could not add {args[1]} to {entityUid}.");
+            }
+        }
+    }
+
+    [AdminCommand(AdminFlags.Debug)]
+    public sealed class RemoveTagCommand : LocalizedCommands
+    {
+        [Dependency] private readonly IEntityManager _entityManager = default!;
+
+        public override string Command => "removetag";
+        public override string Description => "Removes a tag from a given entity";
+        public override string Help => "Usage: removetag <entity uid> <tag>";
```

Comment:
completions would be really nice

Reply:
with the current tagcomp tags for removetag, and `CompletionHelper.PrototypeIDs<TagPrototype>()` for addtag